### ISSUE TITLE
Natural Voices in iOS Text to Speech Not Working

### ISSUE THREAD

**Issue by raynergit** [omnivore-app/omnivore#4421]
Hi there, I'm reopening this item as I had accidentally closed it previously, but there are several users encountering this issue.

From another user, saeedsmaili: "Same issue here for more than a week. Maybe we should open another issue if we can't re-open this one."

_Originally posted by @saeedesmaili in https://github.com/omnivore-app/omnivore/issues/4416#issuecomment-2409222782_

**Comment by saeedesmaili** [comment]
Confirming this issue (it's a follow up from https://github.com/omnivore-app/omnivore/issues/4416 with other users reporting the same issue).

I'm actually getting headache listening to the other tts voices in the app (which aren't as realistic but don't have this issue).

**Comment by vkobel** [comment]
I have the same issue with natural voices :(

**Comment by adarsh** [comment]
The voice not working for me is Shimmer.

**Comment by glidergold** [comment]
I commented on the other thread, but the issue persists. The natural voices don't work for me either. I get load errors trying to read the articles.

**Comment by Shellowb** [comment]
Same issue here!!! We can support with something technically? I really like the app

**Comment by adarsh** [comment]
I wonder if it's this dependency which needs updating? https://github.com/omnivore-app/omnivore/pull/4434

**Comment by saeedesmaili** [comment]
> I wonder if it's this dependency which needs updating? https://github.com/omnivore-app/omnivore/pull/4434

I don't think so, since the voices with this issue (i.e Nova, Shimmer, etc) belong to OpenAI, not Microsoft (but I might be wrong, and maybe Omnivore is using the OpenAI TTS service on Microsoft Azure). How can we make sure the developers of the project are aware of this issue?

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
